server/controllers/helpers/authorise.js

```diff
@@ -3,7 +3,7 @@
 
   module.exports = function(req, document) {
     // User's roles
-    var userRoles = req.decoded._doc.roles;
+    var userRoles = req.user.roles;
```

Comment:
**_Code Quality - Nitpick_**

Using a package like lodash would reduce the number of lines of this function.